src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Contact/Edit.tsx

```diff
@@ -72,6 +73,7 @@ const Edit = ({ onClose, incident, submit }: Props) => {
         <div>
           <StyledInput
             {...register('email')}
+            id={'email'}
```

Comment:
For my knowledge: why should an input have an id?

Reply:
a label in associated with an input by either nesting the input, or <label for='naam'>name</label> <input id='naam' asdfasfa />